### DISCUSSION TITLE
mustang and sally broken

### DISCUSSION THREAD

**Discussion by conbobcob** [nzp-team/nzportable#849] [General]
I noticed when you reload your last bullets in reserve it adds more than 6 bullets into the left gun. Has anybody else noticed this?
![Screenshot 2024-07-19 11 54 14 AM](https://github.com/user-attachments/assets/81bd7517-1a7e-45ca-a9e6-392244e80171)

**Comment by DerpedCrusader** [comment]
If you can consistently reproduce this, could you fill out an issue? 

Make sure to follow the provided template as best you can
